src/sagemaker/jumpstart/types.py

```diff
@@ -763,6 +769,7 @@ def from_json(self, json_obj: Dict[str, Any]) -> None:
             if json_obj.get("default_payloads")
             else None
         )
+        self.private_bucket = json_obj.get("private_bucket", False)
```

Comment:
Question - this means `private_bucket` should be present in the spec file for a gated model? Just wanted to settle on this flag name as I'm also working on MH change that'll make use of the flag.

Reply:
`private_bucket` should equal `True` for gated models

Reply:
Actually, I'm changing `private` --> `gated` so it'll be `gated_bucket`